docs/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md

```diff
@@ -6,27 +6,25 @@ title: UI Server-Side Pagination
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination"/>
 </head>
 
-:::caution
```

Comment:
i've made the comments the wrong way around, but those in `versioned_docs/version-2.12/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md` need to be applied here